web/src/games/frenchTarot/instructions.md

```diff
@@ -0,0 +1,24 @@
+French Tarot is a trick-taking game in which the partnerships vary from hand to
```

Comment:
Just making sure, all this content and the images were authored by you? Asking for copyright reasons

Reply:
The text are extracts from the web site pagat.com - I added a notice on that at the top of this file. Furthermore, I added a copyright notice about the artwork at the end of this file.

Reply:
Ok, the artwork seems fine as it is public domain, but the text we can't copy from other website as they don't have a license that allows it... Can you remove the text for now? We can try to write our own later